core/storage/feedback/gae_models.py

```diff
@@ -271,8 +377,19 @@ def create(cls, exploration_id, thread_id, author_id, exploration_version,
                state_name, description, state_content):
         """Creates a new SuggestionModel entry.
 
-        Throws an exception if a suggestion with the given thread id already
-        exists.
+        Args:
+            exploration_id: str. ID of the corresponding exploration.
+            thread_id: str. ID of corresponding thread.
+            author_id: str. ID of user who submitted the suggestion.
+            state_name: str. ID of the state the thread is for.
+            description: str. Learner provided description of suggestions.
+            state_content: dict. Contains two keys, "type" and "value".
```

Comment:
Start your dev server, create an exploration, and publish it. Log in as a learner, go to the library page, and look for the "Suggest an edit" button in the top right of the navbar. You'll be able to suggest an edit to the content of the exploration that way.

(Note that you need to be logged in, and that this is different from the feedback mechanism.)